<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoTypeSelectDialog</class>
 <widget class="QDialog" name="AutoTypeSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto-Type - KeePassXC</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select an entry to Auto-Type or find an entry using the search. Right click a row for more options.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AutoTypeMatchView" name="view"/>
   </item>
   <item>
    <widget class="QWidget" name="buttonBox_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="filterRadio">
        <property name="text">
         <string>&amp;Filter Matches</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="searchRadio">
        <property name="text">
         <string>&amp;Search Database</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="search">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Filter or Search...</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="action">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutoTypeMatchView</class>
   <extends>QTableView</extends>
   <header>autotype/AutoTypeMatchView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>view</tabstop>
  <tabstop>filterRadio</tabstop>
  <tabstop>searchRadio</tabstop>
  <tabstop>search</tabstop>
  <tabstop>action</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
